src/components/home/Landing.jsx

```diff
@@ -1,7 +1,30 @@
+import Image from "next/image";
+import image1 from "@/public/home/landing.webp";
+import image2 from "@/public/waveBottom.webp";
+import Button from "../Button";
+
 const Landing = () => {
   return (
-    <div className="">
-      <p>Landing</p>
+    <div className="w-screen relative -translate-y-5">
+      <Image src={image1} alt="Background image 1" className="w-screen" />
+      <div className="flex justify-center w-[50%] h-full bg-swim-blue-200/60 top-[0%] absolute">
+        <div className="flex flex-col absolute top-[40%] left-[7%]">
```

Comment:
Combine this divider tag with the divider tag in line 10.  Use padding instead of absolute positioning.